analyses/cell-type-wilms-tumor-06/notebook/04_annotation_Across_Samples_exploration.Rmd

```diff
@@ -0,0 +1,310 @@
+
+The fetal full reference (Cao et al.) provides three levels of annotations:
+
+- `fetal_full_predicted.organ` is one of the 15 organs used for building the reference. 
+We were nicely surprised that, for many of the samples, most of the cells from the Wilms tumor cohort mapped to fetal kidney cells. 
+
+- `fetal_full_predicted.annotation.l1` holds predicted cell type annotation
+
+- `fetal_full_predicted.annotation.l2` holds the combination of the predicted organ and predicted cell type annotation
+
+    
+
+```{r fig.width=25, fig.height=6,  results='asis'}
+tmp <- table(cell_type_df$sample_id, cell_type_df$organ)
+
+knitr::kable(tmp)
+```
+
+```{r fig.width=25, fig.height=6}
+do_Table_Heatmap(first_group = "sample_id", last_group = "organ")
+```
+
+### Predicted compartment
+
+The predicted compartment is the result of the label transfer fron the human fetal kidney atlas: 
+[Stewart et al.](https://www.science.org/doi/10.1126/science.aat5031) created a [human fetal kidney atlas](https://www.kidneycellatlas.org/). 
+This reference contains only fetal kidney cells and has been precisely annotated by kidney experts. 
+The label transfer have been performed in the notebook `02b_fetal_kidney_reference_Stewart_{sample_id}.Rmd`
+
+The fetal kidney reference (Stewart et al.) provides two levels of annotations:
+
+- `fetal_kidney_predicted.compartment` is one of the 4 main compartments composing a fetal kidney.
+We expect immune and endothelial cells to be healthy (non-canerous) cells identified easily with high confidency. 
+We expect stroma and fetal nephron compartment to contain both normal and malignant cells.
+
+    
+    - `immune` cells
+    
+    - `stroma` cells
+    
+    - `fetal_nephron` cells
+    
+    - `endothelial` cells
+    
+
+
+- `fetal_kidney_predicted.cell_type` holds predicted cell type annotations that details further the four compartments, as for example:
+
+    - `immune` subtypes: myeloid cells, lymphocytes, ...
+    
+    - `stroma` subtypes: mesenchymal stem cells, fibroblasts, ...
+    
+    - `fetal_nephron` subtypes: podocytes, uteric bud, mesenchymal cells (that appear to be cap mesenchyme cells) ...
+    
+    - `endothelial` cells
+
+
+We think that the first level of annotation `fetal_kidney_predicted.compartment` can be useful in identifying non-cancer cells, at least the endothelial and immune cells. 
+
+
+```{r fig.width=25, fig.height=3, results='asis'}
+tmp <- table(cell_type_df$sample_id, cell_type_df$compartment)
+
+knitr::kable(tmp)
+```
+
+```{r fig.width=25, fig.height=3}
+
+do_Table_Heatmap(first_group = "sample_id", last_group = "compartment")
+```
+
+
+
+### Evaluate annotations with marker genes
+
+Here we evaluate with marker genes the identification of endothelial, immune cells sub-populations.
+
+
+##### Endothelial cells
+
+We look at the endothelial marker "ENSG00000110799" = "VWF"
+
+```{r fig.height=3, fig.width=15, message=FALSE, warning=FALSE, out.width='100%'}
```

Comment:
This needs to be a lot taller to be able to see. I can't directly suggest on this line with GitHub, but try like 6 inches?